e2e/_suites/fleet/integrations.go

```diff
@@ -17,18 +18,74 @@ const elasticEnpointIntegrationTitle = "Endpoint Security"
 // IntegrationPackage used to share information about a integration
 type IntegrationPackage struct {
 	packageConfigID string          `json:"packageConfigId"`
-	name            string          `json:"name"`
-	title           string          `json:"title"`
-	version         string          `json:"version"`
+	Name            string          `json:"name"`
+	Title           string          `json:"title"`
+	Version         string          `json:"version"`
 	json            *gabs.Container // json representation of the integration
 }
 
+type Policy struct {
+	ID            string             `json:"id,omitempty"`
+	Name          string             `json:"name"`
+	Description   string             `json:"description"`
+	Namespace     string             `json:"namespace"`
+	Enabled       bool               `json:"enabled"`
+	AgentPolicyID string             `json:"policy_id"`
+	OutputID      string             `json:"output_id"`
+	Inputs        []PolicyInput      `json:"inputs"`
+	Package       IntegrationPackage `json:"package"`
+}
+
+type PolicyInput struct {
+	Type    string         `json:"type"`
+	Enabled bool           `json:"enabled"`
+	Streams []interface{}  `json:"streams"`
+	Vars    map[string]Var `json:"vars,omitempty"`
+}
+
+type Var struct {
```

Comment:
The error in golint can be found here: https://beats-ci.elastic.co/blue/organizations/jenkins/e2e-tests%2Fe2e-testing-mbp%2FPR-924/detail/PR-924/1/tests
>e2e/_suites/fleet/integrations.go:27:6: exported type Policy should have comment or be unexported
e2e/_suites/fleet/integrations.go:39:6: exported type PolicyInput should have comment or be unexported
e2e/_suites/fleet/integrations.go:46:6: exported type Var should have comment or be unexported

```suggestion
// Policy represents an integration policy
type Policy struct {
	ID            string             `json:"id,omitempty"`
	Name          string             `json:"name"`
	Description   string             `json:"description"`
	Namespace     string             `json:"namespace"`
	Enabled       bool               `json:"enabled"`
	AgentPolicyID string             `json:"policy_id"`
	OutputID      string             `json:"output_id"`
	Inputs        []PolicyInput      `json:"inputs"`
	Package       IntegrationPackage `json:"package"`
}

// PolicyInput represents an input for the integration policy
type PolicyInput struct {
	Type    string         `json:"type"`
	Enabled bool           `json:"enabled"`
	Streams []interface{}  `json:"streams"`
	Vars    map[string]Var `json:"vars,omitempty"`
}

// Var represents variables in a policy input
type Var struct {
```